lib/doveadm.js

```diff
@@ -23,18 +23,28 @@ export const doveadm_response_types = {
 }
 
 export const doveadm_args_query = {
-	//example: ['mailbox', 'INBOX/myfoldertoo', 'savedbefore', 'since', '30d'],
+	example: ['mailbox', 'INBOX/myfoldertoo', 'savedbefore', 'since', '30d'],
 	positional: true,
 	type: doveadm_arg_types.SEARCH_QUERY,
 	text: `Search query to apply.`,
 }
 
 export const doveadm_args_usermask = {
+	example: 'username',
 	positional: true,
 	type: doveadm_arg_types.STRING,
 	text: `User Mask.`
 }
 
+export const doveadm_args_human_timestamp = `
+Allowable formats:
```

Comment:
Allowed?

Reply:
Allowed means you are giving actual permission to do something.  Allowable describes the things that are permissible (that a user is allowed to use).  So allowable is correct here.